components/HTMLBlock.jsx

```diff
@@ -6,7 +6,11 @@ const PropTypes = require('prop-types');
 const MATCH_SCRIPT_TAGS = /<script\b[^>]*>([\s\S]*?)<\/script>\n?/gim;
 
 const extractScripts = (html = '') => {
-  const scripts = [...html.matchAll(MATCH_SCRIPT_TAGS)].map(m => m[1]);
```

Comment:
We should probably polyfill missing native methods like this in the long run.

Reply:
I don't understand babel, but shouldn't it already be:

```
$ cat .babelrc 
{
  "presets": [
    [
      "@babel/preset-env",
      {
        "targets": {
          "browsers": ["last 2 versions"]
        }
      }
    ],
    "@babel/preset-react"
  ],
  "plugins": [
    "@babel/plugin-proposal-class-properties",
    "@babel/plugin-proposal-object-rest-spread",
    "@babel/plugin-proposal-optional-chaining"
  ]
}
```